back-end/index.js

```diff
@@ -3,31 +3,72 @@ const app = express();
 const mysql = require('mysql');
 const cors = require('cors');
 const axios = require('axios');
+const { json } = require('express');
 
 
+app.use(cors());
+app.use(express.json());
+
 const db = mysql.createPool({
     host: '127.0.0.1',
     user: 'root',
-    password: 'dbpassword',
-    database: 'dbname'
+    password: '88147988rA@',
```

Comment:
Faltou um [.env
](https://www.freecodecamp.org/news/nodejs-custom-env-files-in-your-apps-fa7b3e67abe1/)